rasa/core/run.py

```diff
@@ -180,8 +187,16 @@ def serve_application(
     from rasa.core import broker
 
     try:
-        _, nlu_model = get_model_subdirectories(get_model(model_path))
-        _interpreter = NaturalLanguageInterpreter.create(nlu_model, endpoints.nlu)
+        unpacked_model_context = get_model(model_path)
+        if unpacked_model_context:
+            with unpacked_model_context as unpacked_model:
+                _, nlu_model = get_model_subdirectories(unpacked_model)
+                _interpreter = NaturalLanguageInterpreter.create(
+                    nlu_model, endpoints.nlu
+                )
+        else:
```

Comment:
This was actually needed because `get_model` returns Optional[ContextManager], so the type checker was complaining about `None` not having `__enter__` and `__exit__` attributes. I would like to submit a PR immediately after this one changing the behavior of `get_model` so that it always returns ContextManager, or else throws an exception (instead of using `None` as an error value).